### PR TITLE
IMAGINGCLOUD-88 - Aspose.Imaging v18.6 release integration

### PR DESCRIPTION
**Pull request by MaxCrank**
Removed overkill test cases after moving them to the lower layer, updated README, updated NuGet and nuspec, added temp folder usage for tests, updated Storage SDK.

### CODE REVIEW

File: src/Aspose.Imaging.Cloud.Sdk/Client/ApiException.cs [hunk 1]
```diff
@@ -1,4 +1,6 @@
...

namespace Aspose.Imaging.Cloud.Sdk.Client 
{
    using System;
+
+    /// <summary>
    /// Imaging API Exception
    /// </summary>
  public class ApiException : Exception 
  {
```
Comment (roxboris) [inline src/Aspose.Imaging.Cloud.Sdk/Client/ApiException.cs]: @MaxCrank Please remove damaged comment

File: src/Aspose.Imaging.Cloud.Sdk.Test/Base/ApiTester.cs [hunk 2]
```diff
@@ -117,13 +119,11 @@ public abstract class ApiTester
         protected readonly string[] BasicExportFormats = new string[]
         {
             "bmp",
-            "jpg",
-            "psd",
-            "tiff",
             "gif",
+            "jpg",
             "png",
-            "j2k",
```
Comment (roxboris) [inline src/Aspose.Imaging.Cloud.Sdk.Test/Base/ApiTester.cs]: @MaxCrank Are you sure that need to remove j2k, webp formats from tests?

Comment (MaxCrank) [reply src/Aspose.Imaging.Cloud.Sdk.Test/Base/ApiTester.cs]: @roxboris please check corporate mail regarding tests across all levels.

File: src/Aspose.Imaging.Cloud.Sdk/Client/ApiException.cs (hunk 1 above)
Comment (MaxCrank) [reply src/Aspose.Imaging.Cloud.Sdk/Client/ApiException.cs]: @roxboris fixed line endings in this file for GitHub.

**Pull request merged**